src/mod/acme/acme.go

```diff
@@ -164,15 +173,25 @@ func (a *ACMEHandler) ObtainCert(domains []string, certificateName string, email
 		return false, err
 	}
 
+	// Load certificate info from JSON file
+	certInfo, err := LoadCertInfoJSON(fmt.Sprintf("./conf/certs/%s.json", certificateName))
+	if err == nil {
+		useDNS = certInfo.UseDNS
+		if dnsServers == "" && len(certInfo.DNSServers) > 0 {
```

Comment:
As you have added a new field `DNSservers`, when a user using old cert info config file, the `certInfo.DNSServers` will be `nil`. You will also need to check if `certInfo.DNSServers == nil` before this line.